### DISCUSSION TITLE
Is it possible to move filters outside of the table

### DISCUSSION THREAD

**Discussion by agjs** [TanStack/table#2157] [General]
Hey guys. Thanks for the useful library, saves a lot of time.

I have a question though. Is it possible to render the filters outside of the actual table. More specifically, I need to render filters in the parent component and the table inside of the child component. So essentially is it possible to "decouple" filters from the actual table. 

In all your examples, filters are put inside of the table for each header. 

Thanks in advance!

**Comment by bharathch0404** [comment]
Have the same question
